File: test/fixtures/java-rest-controllers/quarkus-jaxrs-resource.java
package com.example;

import com.example.dto.ItemDto;
import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;
import jakarta.ws.rs.PathParam;

@Path("/v1/items")
public class ItemResource {
    @GET
    @Path("/{id}")
    public ItemDto get(@PathParam("id") String id) {
        return null;
    }
}

File: test/fixtures/kotlin-rest-controllers/quarkus-jaxrs-resource.kt
package com.example

import com.example.dto.ItemDto
import jakarta.ws.rs.GET
import jakarta.ws.rs.Path
import jakarta.ws.rs.PathParam

@Path("/v1/items")
class ItemResource {
    @GET
    @Path("/{id}")
    fun get(@PathParam("id") id: String): ItemDto {
        throw NotImplementedError()
    }
}
